pkg/mqtrigger/metrics.go

```diff
@@ -45,6 +45,20 @@ var (
 		},
 		[]string{"trigger_name", "trigger_namespace", "topic", "partition"},
 	)
+	triggerStatus = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "fission_mqt_trigger_status",
+			Help: "Status of a trigger",
```

Comment:
Status of an individual trigger 1 if processing otherwise 0.

Reply:
Done